### DISCUSSION TITLE
I cant play multiplayer matches bc this appears:

### DISCUSSION THREAD

**Discussion by OsuLee02** [ppy/osu#16671] [Q&A] [answered]
![Screenshot_20220128-010320](https://user-images.githubusercontent.com/98571921/151485956-8db66c96-b2c8-4e34-8c9e-9ae48e0fdc2b.png)

Right before the song is loading this appears (up on the right.) and then Im transfered to the multiplayer room lobby that Im in. What's causing this? :(

**Comment by peppy** [comment, marked as answer]
What platform and what release? You may need to re-download if you are running on one of the revoked releases as they won't work online.

**Reply by peppy** [reply]
I've added back the revoked hashes as the revoked builds were only an issue for some users and should probably still work. Please try once more.

**Reply by OsuLee02** [reply]
Its working now! Thxx
